eq-author-api/schema/typeDefs.js

```diff
@@ -98,12 +98,21 @@ type DeletedQuestionnaire {
   id: ID!
 }
 
+type Folder {
+  id: ID!
+  alias: String
+  enabled: Boolean!
+  pages: [Page]
+  skipConditions: [ExpressionGroup2]
+  position: Int
```

Comment:
I think position should be non-null to be consistent with the other objects which have position:

```suggestion
  position: Int!
```